decidim-core/lib/tasks/decidim_webpacker_tasks.rake

```diff
@@ -46,8 +49,34 @@ namespace :decidim do
       FileUtils.rm(path, force: true)
     end
 
+    def add_binstub_load_path(binstub_path)
+      file = rails_app_path.join(binstub_path)
+
+      contents = ""
+      File.readlines(file).each do |line|
+        contents += line
+        next unless line =~ %r{^require "bundler/setup"$}
+
+        contents += "\n"
+        contents += "# Add the Decidim override load path to override webpacker functionality\n"
+        contents += "$LOAD_PATH.unshift \"\#{Gem.loaded_specs[\"decidim-core\"].full_gem_path}/lib/gem_overrides\"\n"
```

Comment:
💬 Is it possible to make this method idempotent? In fact, was the rake task idempotent after this change? Does this make sense for you?

I'd not like to have binstubs with many duplicated lines if I run the webpacker install task several times, but I don't know how much effort would this require.

Reply:
@leio10 This is fixed.